content/en-US/api/fontStretch.md

```diff
@@ -0,0 +1,67 @@
+---
+recipe: api-interface
+title: 'CanvasRenderingContext2D.fontStretch'
+mdn_url: /en-US/docs/Web/API/CanvasRenderingContext2D/fontStretch
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.CanvasRenderingContext2D.fontStretch
+---
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/textLetterSpacing](https://developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/textLetterSpacing).**
+
+## Description
+
+The `fontStretch` property of the `CanvasRenderingContext2D` interface
+allows the developers to select different font-face from a font. The default
+value is `normal`.
+
+## Syntax
+
+`CanvasRenderingContext2D.fontStretch = "ultra-condensed"`
+`CanvasRenderingContext2D.fontStretch = "extra-condensed"` 
+`CanvasRenderingContext2D.fontStretch = "condensed"`
+`CanvasRenderingContext2D.fontStretch = "semi-condensed"` 
+`CanvasRenderingContext2D.fontStretch = "normal"` 
+`CanvasRenderingContext2D.fontStretch = "semi-expanded"`
+`CanvasRenderingContext2D.fontStretch = "expanded"`
+`CanvasRenderingContext2D.fontStretch = "extra-expanded"`
+`CanvasRenderingContext2D.fontStretch = "ultra-expanded"`
+`var fontStretch = CanvasRenderingContext2D.fontStretch`
```

Comment:
Maybe do this as I did the `fontKerning` syntax. There may be examples of syntax done like this in MDN, but it is redundant with other things on the page.